op-supervisor/supervisor/backend/rewinder/rewinder_test.go

```diff
@@ -770,7 +771,7 @@ func setupTestChains(t *testing.T, chainIDs ...eth.ChainID) *testSetup {
 	require.NoError(t, err)
 
 	// Create ChainsDB with mock emitter
-	chainsDB := db.NewChainsDB(logger, depSet, metrics.NoopMetrics)
+	chainsDB := db.NewChainsDB(logger, depSet, metrics.NoopMetrics, params.MessageExpiryTimeSecondsInterop)
```

Comment:
Passing it in as yet another constructor arg does not scale when we add more configuration. See other comment about adding it to the dependency set configuration instead. Or if the dependency set is not the right thing either to maintain this parameter, then we can consider creating an interop protocol config struct (and if so, still expose via interface, I regret making rollup.Config used as struct).

Reply:
I've removed the ctor arg. It's all handled by the depset now. I'll remove the override field from the rollup.Config in another PR.